src/edu/wright/cs/raiderplanner/util/Utility.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2018

```

Comment:
Please add proper copyright attribution to the author(s) here.

Reply:
I was thinking that the Utility Class could house multiple functions, including the database creation later down the road. Or would it be better practice to just add different classes in the util folder for future functionality?